Week1/homework/app.js

```diff
@@ -1,11 +1,139 @@
+      author: "Alexander Pushkin",
+    },
+    'Eugene_Onegin': {
+      title: "Eugene Onegin",
+      language: "Russian",
+      author: "Alexander Pushkin",
+    },
+    'The_Cherry_Orchard': {
+      title: "The Cherry Orchard",
+      language: "Russian",
+      author: "Anton Chekhov",
+    },
+    'Evenings_on_a_Farm_Nea_Dikanka': {
+      title: "Evenings on a Farm Nea Dikanka",
+      language: "Russian",
+      author: "Nikolai Gogol",
+    },
+    'Three_Men_in_a_Boat': {
+      title: "Three Men in a Boat",
+      language: "English",
+      author: "Jerome K. Jerome",
+    },
+    'NotreDame_d_Paris': {
+      title: "Notre-Dame de Paris",
+      language: "French",
+      author: "Victor Hugo", 
+    },
+    'The_Shining': {
+      title: "The Shining",
+      language: "English",
+      author: "Stephen King"
+    },
+    'It': {
+      title: "It",
+      language: "English",
+      author: "Stephen King"
+    }
+  };
+  const setImages = {
+    'The_Lord_of_the_Rings': 'images/The_Lord_of_the_Rings.jpg', 
+    'Harry_Potter_and_The_Chamber_of_Secrets': 'images/Harry_Potter_and_The_Chamber_of_Secrets.jpg',
+    'Harry_Potter_and_the_Order_of_the_Phoenix': 'images/Harry_Potter_and_the_Order_of_the_Phoenix.jpg',
+    'The_Master_and_Margarita': 'images/The_Master_and_Margarita.jpg',
+    'The_Little_Tragedies': 'images/The_Little_Tragedies.jpg',
+    'Eugene_Onegin': 'images/Eugene_Onegin.jpg',
+    'The_Cherry_Orchard': 'images/The_Cherry_Orchard.jpg',
+    'Evenings_on_a_Farm_Nea_Dikanka': 'images/Evenings_on_a_Farm_Nea_Dikanka.jpg',
+    'Three_Men_in_a_Boat': 'images/Three_Men_in_a_Boat.jpg',
+    'NotreDame_d_Paris': 'images/NotreDame_d_Paris.jpg',
+    'The_Shining': 'images/The_Shining.jpg',
+    'It': 'images/It.jpg'
+  };
 
-
-  // Replace with your own code
+/*
   console.log(bookTitles);
+*/
+createBookList()
+function createBookList() {
+  let myMainDiv = document.getElementById('main');
+  let myH1 = document.createElement('h1');
+  myMainDiv.appendChild(myH1);
+  let pageTitile = document.createTextNode("My favorite books");
+  myH1.appendChild(pageTitile);
+  let myUl = document.createElement('ul');
+  myMainDiv.appendChild(myUl);
+  for (let i = 0; i < bookTitles.length; i++) {
+    let myLi = document.createElement('li');
+    myUl.appendChild(myLi);
+    let bookName = document.createTextNode(Object.values(booksInformation)[i].title);
+    let bookAuthor = document.createTextNode("The book is written by " + Object.values(booksInformation)[i].author);
+    let booklanguage = document.createTextNode("The book is written in " + Object.values(booksInformation)[i].language);
+    let myH2 = document.createElement("h2");
+    let myPtag1 = document.createElement("p");
+    let myPtag2 = document.createElement("p");
+    myH2.appendChild(bookName);
+    myPtag1.appendChild(bookAuthor);
+    myPtag2.appendChild(booklanguage);
+    myLi.innerHTML += myH2.outerHTML + myPtag1.outerHTML + myPtag2.outerHTML;
```

Comment:
it would be better to append these tags to the `li` tag you created `innerHTML` have some issues so you can use it but it's better to stay away of it.